app/assets/stylesheets/flash.scss

```diff
@@ -0,0 +1,21 @@
+.flash-error {
+  color: #ff0000;
+  font-size: 18px;
+  font-weight: bold;
+  margin-top: 1em;
+  text-align: center;
+}
+
+.flash-notice {
+  color: #008000;
+  font-size: 18px;
+  margin-top: 1em;
+  text-align: center;
+}
+
+.flash-success {
+  text-align: center;
+  color: #008000;
```

Comment:
Color literals like `#008000` should only be used in variable declarations; they should be referred to via variable everywhere else.